modules/payment/tests/src/FunctionalJavascript/PaymentCheckoutTest.php

```diff
@@ -57,7 +57,7 @@ class PaymentCheckoutTest extends CommerceBrowserTestBase {
    */
   protected function getAdministratorPermissions() {
     return array_merge([
-      'administer profiles',
+      'administer profile',
```

Comment:
I think that this change might already be included in the latest commerce dev, on a different line, causing it not to apply cleanly. The change to CheckoutFlowBase applies just fine though.